workflow/rules/common.smk

```diff
@@ -60,11 +60,25 @@ if config.get("trimmer_software", "None") == "fastp_pe":
         "prealignment/fastp_pe/{sample}_{flowcell}_{lane}_{barcode}_{type}_fastq1.fastq.gz",
         "prealignment/fastp_pe/{sample}_{flowcell}_{lane}_{barcode}_{type}_fastq2.fastq.gz",
     ]
+    alignment_input_fq1 = lambda wilcards: [
+        "prealignment/fastp_pe/{sample}_{flowcell}_{lane}_{barcode}_{type}_fastq1.fastq.gz",
+    ]
+    alignment_input_fq2 = lambda wilcards: [
+        "prealignment/fastp_pe/{sample}_{flowcell}_{lane}_{barcode}_{type}_fastq2.fastq.gz",
+    ]
 elif config.get("trimmer_software", "None") == "None":
     alignment_input = lambda wildcards: [
         get_fastq_file(units, wildcards, "fastq1"),
         get_fastq_file(units, wildcards, "fastq2"),
     ]
+    alignment_input_fq1 = lambda wilcards: [
+        get_fastq_file(units, wildcards, "fastq1"),
+    ]
+    alignment_input_fq2 = lambda wilcards: [
+        get_fastq_file(units, wildcards, "fastq2"),
+    ]
```

Comment:
this is weird, why doing a `lambda` expression if there is nothing happening with wildcards?

Reply:
Same for the others. It can probably just be 

```
alignment_input_fq2 = [alignment_input[1]]
```

However, I think this can be simplified even more.